cassandra/connection.py

```diff
@@ -813,6 +830,8 @@ def _connect_socket(self):
                     self._socket = self._ssl_impl.wrap_socket(self._socket, **self.ssl_options)
                 self._socket.settimeout(self.connect_timeout)
                 self._initiate_connection(sockaddr)
+                port = self._socket.getsockname()[1]
+                print(f'connection ({id(self)}) port={port} should be shard_id={port % self.total_shards}')
```

Comment:
```suggestion
                log.debug("Connection (%s) port=%s should be shard_id=%s" % (id(self), port, port % self.total_shards))
```

Reply:
FYI if using logging, it's best to not format
```python
log.debug("Connection (%s) port=%s should be shard_id=%s", id(self), port, port % self.total_shards)
```

it saves lot of cycles when running without debug, I've found some issues like that in boto3, jumping right at me when trying to profile it.